api/src/main/java/org/openmrs/module/pihcore/apploader/apps/patientregistration/SectionsSierraLeone.java

```diff
@@ -128,7 +128,7 @@ private Question getLocalAddressQuestion() {
         // If there are address hierarchy levels configured, use the address hierarchy widget, otherwise use the standard address widget
         List<AddressHierarchyLevel> levels = Context.getService(AddressHierarchyService.class).getAddressHierarchyLevels();
         if (levels != null && levels.size() > 0) {
-            //q.setDisplayTemplate(getAddressHierarchyDisplayTemplate(levels));
+            q.setDisplayTemplate(getAddressHierarchyDisplayTemplate(levels));
             f.setWidget(getAddressHierarchyWidget(levels, getLocalContactAddressFieldMappings(), true));
```

Comment:
Now that the result of `getAddressHierarchyDisplayTemplate(levels)` is able to be customized based on `isShowCountryInAddressHiercharchy()`, do we need this custom implementation of this method for Sierra Leone?  Is there a reason we can't uncomment this for everything?  Do we know why this was commented out in the first place and can we test that this works as expected (look at the git blame / git history and see what tickets are associated with this)?

Reply:
Sorry @mseaton, I am not sure I understand the question here. The q.setDisplayTemplate(getAddressHierarchyDisplayTemplate(levels)) line has already been uncommented for Sierra Leone. 
Are you asking to uncomment this line in the SectionsHaiti.java class? That's the only other place I found this line to be commented out. I think it is very hard to know the history of this line. It looks like this was part of a massive refactoring effort when you moved code from the mirebalais module into the pihcore?

<img width="1327" alt="Screenshot 2023-10-10 at 12 21 35 AM" src="https://github.com/PIH/openmrs-module-pihcore/assets/1633285/8acea8c9-0891-4933-8bea-1650bc98915a">

Reply:
Sorry, I misread what was happening here.  This isn't the main address question, but a custom "local address" question that you added back in June.  I assume you just copied this from the Haiti version, with the commented out line in it?  Makes sense to uncomment it now like you have done, so I think this is good. But if we copied this from somewhere (eg. Haiti) where this line is commented out, might be worth revisiting why it is commented out there and whether that is still valid given the improvements you've made here.